Source/Core/Core/HW/WiimoteCommon/WiimoteHid.h

```diff
@@ -21,6 +21,44 @@ constexpr u8 HID_HANDSHAKE_SUCCESS = 0;
 constexpr u8 HID_PARAM_INPUT = 1;
 constexpr u8 HID_PARAM_OUTPUT = 2;
 
+class HIDWiimote
+{
+public:
+  using InterruptCallbackType = std::function<void(u8, const u8* data, u32 size)>;
+
+  virtual ~HIDWiimote() = default;
+
+  virtual void EventLinked() = 0;
+  virtual void EventUnlinked() = 0;
+
+  // Every ~200hz after HID channels are established.
```

Comment:
Tiny nit, but this comment feels incomplete. Should it be more like "Occurs every ~200hz [...]" or "Called every ~200hz [...]"?